From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: AnotherZane <zanedpereira2@gmail.com>
Date: Fri, 7 Aug 2020 20:17:51 +0530
Subject: [PATCH] Sample Patch


diff --git a/src/main/java/net/minecraft/client/Minecraft.java b/src/main/java/net/minecraft/client/Minecraft.java
index 1641c29fe81c9fc834f0d8b8dbd4947b25cd4c56..0d6e9ad71eb1ae905c15fbab8c1a51b26e242cab 100644
--- a/src/main/java/net/minecraft/client/Minecraft.java
+++ b/src/main/java/net/minecraft/client/Minecraft.java
@@ -355,6 +355,8 @@ public class Minecraft extends RecursiveEventLoop<Runnable> implements ISnooperI
    private IProfileResult field_238174_aV_;
    private String debugProfilerName = "root";
 
+   // This is a sample comment from MCP-Patcher! Happy developing!
+
    public Minecraft(GameConfiguration gameConfig) {
       super("Client");
       instance = this;
